### DISCUSSION TITLE
Windows with SSH: Permission denied when pushing to GitHub

### DISCUSSION THREAD

**Discussion by btbristow** [Vinzent03/obsidian-git#619] [Problems]
Hi, I’m using Obsidian Git on Windows and getting a permission error when I push, as follows:

`Permission denied (publickey) Fatal: Could not read from remote repository`

- I can commit and push in Git Bash after being prompted for my SSH passphrase.
- I **don't** get prompted to enter my SSH passphrase when I push to GitGhub in Obsidian, so I can never authenticate.
- The plugin docs for [Windows authentication](https://publish.obsidian.md/git-doc/Authentication#Windows) say: "Just execute any authentication command like push/pull/clone and a pop window should come up." 
- They also say to make sure Git Credential Manager is running, and it is. If I run `git config credential.helper`, I get `manager`.

*Edit:*
I've done more research and see that the plugin doesn't support SSH for Windows. Can this be added to the docs? Other than that, I really appreciate your work and am using it daily. Thanks!

**Comment by paguasmar** [comment]
I would advise to try the following tutorial:
https://www.youtube.com/watch?v=a-zX_qc2S-M

Note: Do not add a passphrase since that will slow you done
